oslo_vmware/rw_handles.py

```diff
@@ -332,6 +355,7 @@ def update_progress(self):
         self._last_progress_udpate = now
         progress = int(self._get_progress())
         self._log_progress(progress)
+        self._check_progress_stalled(progress)
```

Comment:
Looks like you're missing the second parameter (`now`).

Reply:
yes. thank you.